packages/openssl.rb

```diff
@@ -47,7 +47,7 @@ def self.build
       CFLAGS=\"#{@ARCH_C_LTO_FLAGS}\" CXXFLAGS=\"#{@ARCH_CXX_LTO_FLAGS}\" \
       LDFLAGS=\"#{@ARCH_LDFLAGS}\" \
       mold -run ./Configure --prefix=#{CREW_PREFIX} \
-      --libdir=#{CREW_LIB_PREFIX} \
+      --libdir=#{ARCH_LIB} \
```

Comment:
So I'm guessing they use `prefix` and `libdir` together to get the actual library path?

Reply:
In the last six months they broke their previous behavior.... I've opened an issue, which reopens an earlier issue...